File: src/order-processor-service/src/main/java/com/gabrielaraujo/order_processor_service/adapters/dao/tables/ClientTable.java
package com.gabrielaraujo.order_processor_service.adapters.dao.tables;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "tb_clients")
@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ClientTable {
    @Id
    @Column(name = "id_client")
    private Integer id;

    @Column(name = "tx_name")
    private String name;

    @Column(name = "tx_description")
    private String description;
}

File: src/order-analytics-api/OrderAnalytics.Adapters/DAO/Tables/ClientTable.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace OrderAnalytics.Adapters.DAO.Tables;

[Table(("tb_clients"))]
public class ClientTable
{
    [Key]
    [Column("id_client")]
    public int Id { get; set; }
    
    [Column("tx_name")]
    public string? Name { get; set; }
    
    [Column("tx_description")]
    public string? Description { get; set; }
}